src/dateUtil.js

```diff
@@ -0,0 +1,13 @@
+let moment = require('moment-timezone')
+let datemath = require('@elastic/datemath')
+
+let parseAndShift = (exp, timezone) => {
```

Comment:
This is moved to a separate file since I am assuming this facet would also need to deal with all the date UTC nonsense so there might have to be other functions moved here as well so they could be reused...